core/domain/improvements_domain.py

```diff
@@ -194,11 +180,8 @@ def to_dict(self) -> TaskEntryDict:
             'target_id': self.target_id,
             'issue_description': self.issue_description,
             'status': self.status,
-            'resolver_username': (
-                resolver_settings and resolver_settings.username),
-            'resolver_profile_picture_data_url': (
-                resolver_settings and
-                resolver_settings.profile_picture_data_url),
+            'resolver_username': None,
+            'resolver_profile_picture_data_url': None,
```

Comment:
Don't use `None`, just don't add it at all.

Reply:
MyPy type checks flag an error here for missing keys in dict. So I think these need to be added.

Reply:
I think this one does need to be added, yeah.

Reply:
I think it is fine to add these as `None`, i.e. keep the code as is